packages/js/src/workouts/tailwind-components/indexation-step.js

```diff
@@ -0,0 +1,101 @@
+import { useCallback, useState, Fragment } from "@wordpress/element";
+import { addLinkToString } from "../../helpers/stringHelpers.js";
+import { __, sprintf } from "@wordpress/i18n";
+import { ReactComponent as WorkoutStartImage } from "../../../images/motivated_bubble_woman_1_optim.svg";
+import PropTypes from "prop-types";
+import Alert from "./alert";
+import { ConfigurationIndexation } from "./ConfigurationIndexation";
+
+import AnimateHeight from "react-animate-height";
```

Comment:
I would organise a bit better the imports as per our guidelines:
- first external imports
- then internal (Yoast) imports